python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -182,10 +234,72 @@ class ForcedPhotCcdTask(ForcedPhotImageTask):
     """
 
     ConfigClass = ForcedPhotCcdConfig
-    RunnerClass = lsst.pipe.base.ButlerInitializedTaskRunner
+    RunnerClass = pipeBase.ButlerInitializedTaskRunner
     _DefaultName = "forcedPhotCcd"
     dataPrefix = ""
 
+    def adaptArgsAndRun(self, inputData, inputDataIds, outputDataIds, butler):
+        # Filtering the reference catalog is currently handled by
+        # Gen2 specific methods.  To get around this, port in code
```

Comment:
We've been trying to keep `adaptArgsAndRun` cleaner than `runDataRef`. Could you move these 3 steps into a method called `filterReferences()` with a docstring explaining that the old `fetchReferences()` did some filtering in addition to the fetching? (If that accurately reflects why this is necessary of course)

Reply:
And if the code is duplicated elsewhere, a note in the docstring about where to look would help future devs.